mod/zaudio/views/default/js/elgg/zaudio.js

```diff
@@ -7,7 +7,7 @@ define(function (require) {
 	// for unique IDs
 	var i = 0;
 
-	AudioPlayer.setup(elgg.get_site_url() + "mod/zaudio/audioplayer/player.20150521.swf", {width: 290});
+	AudioPlayer.setup(elgg.get_simplecache_url('AudioPlayer.swf'), {width: 290});
```

Comment:
If you use require.toUrl, you can get rid of the `elgg` dependency entirely from this file, which seems like a Good Thing.